.travis.yml

```diff
@@ -1,19 +1,18 @@
 language: php
 
 php:
-  - 5.6
   - 7.0
-  - 7.1
+  - 7.3
 
 env:
   - SYMFONY_VERSION=2.7.*
-  - SYMFONY_VERSION=2.8.*
   - SYMFONY_VERSION=3.0.*
-  - SYMFONY_VERSION=3.1.*
-  - SYMFONY_VERSION=3.2.*
+  - SYMFONY_VERSION=3.4.*
+  - SYMFONY_VERSION=4.0.*
+  - SYMFONY_VERSION=4.2.*
 
 before_script:
-  - composer require symfony/framework-bundle:${SYMFONY_VERSION}
-  - composer require symfony/console:${SYMFONY_VERSION}
+  - composer update symfony/console:${SYMFONY_VERSION}
+  - composer update symfony/framework-bundle:${SYMFONY_VERSION}
```

Comment:
I'm still not sure if we really need to test against all five symfony versions, rather than just the highest and lowest allowed by the composer.json constraints. If we didn't we could make the before_script simpler. But it doesn't do any harm, and the build is only taking about 3½ minutes and we're presumably getting the build time free from Travis, so think this is OK as it is.